tla-types/src/test/scala/at/forsyte/apalache/tla/typecomp/TestFunBuilder.scala

```diff
@@ -585,4 +659,30 @@ class TestFunBuilder extends BuilderTest {
 
   }
 
+  test("debug") {
```

Comment:
Is this a left-over? I can't tell from the test name what this does.

Reply:
Oh, good catch, forgot to remove that.